junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/ParallelExecutionConfiguration.java

```diff
@@ -60,4 +61,17 @@
 	 */
 	int getKeepAliveSeconds();
 
+	/**
+	 * Get the predicate called when the {@code ForkJoinPool} is saturated.
+	 * This occurs when the ForkJoinPool has reached MaxPoolSize
+	 * and there are no worker threads available. The predicate
+	 * should return true if the calling thread should block.
+	 * If the predicate is null or returns false a RejectedExcecutionException
+	 * is thrown.
```

Comment:
I wouldn't go into details here but link to `ForkJoinPool`.

```suggestion
	 * Get the saturate predicate to be used for the execution's {@link ForkJoinPool}.
```

Reply:
Done.